src/pyscript/cli.py

```diff
@@ -1,21 +1,34 @@
 """The main CLI entrypoint and commands."""
 import sys
-import time
-import webbrowser
-from pathlib import Path
 from typing import Any, Optional
 
 from pluggy import PluginManager
 
 from pyscript import __version__, app, console, plugins, typer
-from pyscript._generator import file_to_html, string_to_html
 from pyscript.plugins import hookspecs
 
-DEFAULT_PLUGINS = ["create", "delete"]
+DEFAULT_PLUGINS = ["create", "wrap"]
 
 
-def _print_version():
-    console.print(f"PyScript CLI version: {__version__}", style="bold green")
+def ok(msg=None):
+    """
+    Simply prints "OK" and an optional message, to the console, before cleanly
+    exiting.
+
+    Provides a standard way to end/confirm a successful command.
+    """
+    console.print(f"OK. {msg}", style="green")
```

Comment:
I think this would print `"OK. None"` if no `msg` was provided. We should probably change the default to:
```
def ok(msg: str = ""):
```

and then to be pedantic about trailing whitespace:
```
console.print(f"OK. {msg}".rstrip(), style="green")
```